libs/application/templates/parental-leave/src/hooks/useEstimatedPayments.ts

```diff
@@ -0,0 +1,24 @@
+export const useEstimatedPayments = () => {
+  /*
+  TODO: Bring back payment calculation info, once we have an api
+  https://app.asana.com/0/1182378413629561/1200214178491335/f
+
+  const dob = getExpectedDateOfBirth(application)
+  const { data, error, loading } = useQuery(getEstimatedPayments, {
+    variables: {
+      input: {
+        dateOfBirth: dob,
+        period: [
+          {
+            from: '2021-01-01',
+            to: '2021-01-01',
+            ratio: 100,
+            approved: true,
+            paid: true,
+          },
+        ],
+      },
+    },
+  })
+  */
+}
```

Comment:
Maybe this file should be removed completely while not in use.
Than added later when this is implemented again.

Reply:
Yes I was not sure, but you are right, and the API might might be different as well so I will do that 👍